client/app/telecom/telephony/billingAccount/administration/addGroup/telecom-telephony-billing-account-administration-add-group.html

```diff
@@ -44,7 +44,7 @@ <h1 data-translate="telephony_add_group_title"></h1>
 
                 <div data-ng-if="AddGroupCtrl.order">
                     <p>
-                        <i class="ovh-font ovh-font-success text-success h3 mb-0 mr-2"></i>
+                        <i class="ovh-font ovh-font-success text-success mb-0 mr-2"></i>
```

Comment:
A `<oui-message>` would be great https://master.ui-kit.ovh/#!/oui-angular/message :)

Because this is a hotfix, It's ok for me.